### PR TITLE
Adding Contact Form

### PR DESCRIPTION
**Pull request by Wolfmikerson**
- Add a contact form to index.html
- Style the contact form wt CSS styling
- Verify the data was sent properly

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,34 +1,876 @@
...
+@media (min-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  .portfolio,
+  .about,
+  .contact {
+    display: flex;
+  }
```
Comment (christianbiring1) [inline css/style.css]: - [ ]  Kindly consider removing this block of code as it doesn't make your codebase looks clean

File: css/style.css [hunk 2]
```diff
@@ -1,34 +1,876 @@
...
+}
+
+.contactme {
+  display: flex;
+  justify-content: center;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+}
```
Comment (christianbiring1) [inline css/style.css:396]: [OPTIONAL]
- [ ] Kindly make sure your page is responsive on small-screen devices for accessibility purposes
Your design :point_down: 
image

image
image


Figma template :point_down: 
image

File: index.html [hunk 3]
```diff
@@ -30,9 +33,13 @@ <h2 class="support-text-2 default-custom-fonts lets-text-font">LETS CONNECT</h2>
...
     <section class="grid-container">
-      <div class="flex-item">
-        <img class="image" src="images/Portfolio.svg" alt="Tonic">
+      <div class="grid-item grid-item1 flex-item g1">
+        <img class="image tonicA" src="images/Portfolio.svg" alt="Tonic">
+        <img class="image desktop" src="images/desaktop-2.svg" alt="desktop">
+      </div>
+      <div class="grid-item grid-item2 flex-item g1">
         <p class="tonic">Tonic</p>
         <ul class="list-item default-list-style">
```
Comment (christianbiring1) [inline index.html:44]: [OPTIONAL]
- [ ] For accessibility purposes, kindly make sure all your card titles as grabbed in a header tag `hx` and not a paragraph as is the case in your code

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Wolfmikerson , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good commit messages :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] `I HIGHLY RECOMMEND YOU IMPLEMENT THEM FOR THEY WIL TAKE YOUR PROJECT TO THE NEXT LEVEL` :rocket: 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (Owusu-Desmond) [inline css/style.css]: - [ ] As requested by the previous code reviewer, you should consider removing this block of code as it doesn't make your codebase looks clean. You can read this article for more details telling the reason why committing commented code is bad. [Please, don't commit commented-out code - Kent C. Dodds](https://kentcdodds.com/blog/please-dont-commit-commented-out-code)

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Wolfmikerson  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented
:heavy_check_mark: Good PR description 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kamba56 — COMMENTED** [review]
Hi  @Wolfmikerson, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that **Does not run linters checks **. Kindly make sure that all linters checks are running and submit for a re-review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Wolfmikerson  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**